lib/src/nestable.component.ts

```diff
@@ -596,6 +601,14 @@ export class NestableComponent implements OnInit, OnDestroy {
           this._placeholder
         );
 
+      const index = Array.prototype.indexOf.call(this._placeholder.parentElement.children, this._placeholder);
+
+      if (this._dragIndex === index) {
+        if (this._oldListLength.length === this.list.length) {
+          changedElementPosition = true;
+        }
```

Comment:
if (this._dragIndex === index && this._oldListLength.length === this.list.length) {
  changedElementPosition = true;
}